blockdag/dag.go

```diff
@@ -816,20 +816,51 @@ func (dag *BlockDAG) LastFinalityPointHash() *daghash.Hash {
 	return dag.lastFinalityPoint.hash
 }
 
+// isInSelectedParentChain returns whether aNode is in the selected parent chain of bNode.
+func (dag *BlockDAG) isInSelectedParentChain(aNode, bNode *blockNode) (bool, error) {
+	// Because reachabilityInterval.isAncestorOf and isInSelectedParentChain
```

Comment:
If interval.isAncestorOf is exclusive (a.k.a returns false if aNode==b.Node, if I understand correctly what you mean), why do we need to check this explicitly?

And if it's not exclusive in the sense I defined here, didn't we agree we should make it such?